packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -54,7 +53,7 @@
   $: schema = getSchemaForTable(tableId, { searchableSchema: true }).schema
   $: schemaFields = Object.values(schema || {})
 
-  const onChange = debounce(async function (e, key) {
+  const onChange = async (e, key) => {
```

Comment:
This should probably work fine as long as we don't have rapid saves in succession. In the design section I made a wrapper function that ensures multiple calls to async functions are fired in sequence, letting the first one complete before continuing. The reasoning was that if you had a bad ping, and made 2 changes quickly, the second one would be sent up referencing stale data as the change in the first request hadn't completed yet. Might not be a problem here at all - but if it ever becomes a problem then the change is as easy as:
```js
import { Utils } from "@budibase/frontend-core"

const onChange = Utils.sequential(async (e, key) => {
...
})
```

Reply:
Updated to use this! Thanks!